src/elements/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,47 @@
+import moment from "moment"
+import React from "react"
+import styled from "styled-components"
+
+import { color, space } from "../../helpers"
+
+const ProgressBarBackground = styled.div`
+  height: 3px;
+  background-color: ${color("black10")};
+  margin: ${space(0.5)}px 0 ${space(1)}px;
+  display: flex;
+  justify-content: flex-start;
+`
+
+export const ProgressBar: React.SFC<{
+  currentTime: string
+  countdownStart: string
+  countdownEnd: string
+  highlight: Parameters<typeof color>[0]
+}> = ({
+  currentTime,
+  countdownStart,
+  countdownEnd,
+  highlight = "purple100",
+}) => {
+  const secondsRemaining = moment(countdownEnd).diff(
+    moment(currentTime),
+    "seconds"
+  )
+  const totalSeconds = moment(countdownEnd).diff(
+    moment(countdownStart),
+    "seconds"
+  )
+  const progress = Math.max(0, (secondsRemaining * 100) / totalSeconds)
+
+  return (
+    <ProgressBarBackground>
+      <div
+        style={{
+          transition: "width 0.34s ease",
+          backgroundColor: color(highlight as any),
+          width: progress + "%",
+        }}
+      />
+    </ProgressBarBackground>
+  )
+}
```

Comment:
Would you mind updating this a bit so that there are two components exported from here, allowing devs who are unconcerned about time to be able to simply pass in `pctProgress`? That way it can be detatched from the time mechanism. 

1) `ProgressBar` (which is static, and takes `highlight` and `pctComplete`) 
2) `ProgressBarTimer` (which composes `ProgressBar`, containing time functionality)

Reply:
Sure! Sounds good :+1:

Reply:
Updated